exec/java-exec/src/main/resources/rest/profile/profile.ftl

```diff
@@ -587,6 +622,49 @@
       if (e.target.form) 
         <#if model.isOnlyImpersonationEnabled()>doSubmitQueryWithUserName()<#else>doSubmitQueryWithAutoLimit()</#if>;
     });
+
+    // Convert scientific to Decimal [Ref: https://gist.github.com/jiggzson/b5f489af9ad931e3d186]
+    function scientificToDecimal(num) {
```

Comment:
Won't this be are convenient to do in Java and just pass expected string representation to UI?

Reply:
Ideally, yes.

But the way we construct the `Operators Overview` table is by only injecting values (which then get converted to human readable formats based on data types). That makes it difficult for me to inject these `<div>` elements during construction.

Also, with Javascript, I reduce the processing and construction overhead on the WebServer and have the client browsers do this formatting.

In theory, the entire profile can be constructed by Javascript from the JSON profile, but basic summaries are easier to implement (and faster to calculate) in Java.